z/zabbix/zabbix_ubi_8.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : Zabbix
+# Version       : master
+# Source repo   : https://github.com/zabbix/zabbix.git
+# Tested on     : UBI 8.5
+# Language      : PHP
```

Comment:
@vaishnavipatil3 should this be PHP?

Reply:
I thought it is php seeing languages on github page. Can you please help me with it?

Reply:
yes, so it isn't all in PHP, its split almost 50-50 between PHP and C, @kandarpamalipeddi  can we have a comma separated list of languages here?